css/main.css

```diff
@@ -375,6 +375,10 @@ li {
     height: 4rem;
   }
 
+  .iconz {
+    color: #212121;
```

Comment:
Looks like you couldn't change the color of the icons, make sure to kindly change it :mag: 

Remember that `color` is used to change text colors, you can change the image to make it the correct color, or use an online tool like [icons8](https://icons8.com/) to get the right icon and color! :wink: